app/src/main/java/org/fossasia/openevent/app/module/organizer/detail/OrganizerDetailFragment.java

```diff
@@ -67,6 +74,36 @@ public void onStop() {
         refreshLayout.setOnRefreshListener(null);
     }
 
+    @Override
+    public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
+        inflater.inflate(R.menu.menu_organizer, menu);
+        super.onCreateOptionsMenu(menu, inflater);
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+
+        switch (item.getItemId()) {
+            case R.id.action_change_password:
+                openChangePasswordFragment();
+                break;
+            default:
+                super.onOptionsItemSelected(item);
+                break;
+        }
+        return super.onOptionsItemSelected(item);
+    }
+
+    public void openChangePasswordFragment() {
+        binding.appBar.setVisibility(View.GONE);
+        binding.swipeContainer.setVisibility(View.GONE);
```

Comment:
Why do we need to hide these?

Reply:
Sorry, i forgot to remove this. Actually previously, due to the the Organizer Activity layout bothe the fragments( Organizer detail and change password) were getting overlapped because of which i need to hide Organizer detail fragment once Change Password fragment was opened but then i changed the Organizer detail layout to correct this view hiding and showing such that the app no longer needs these. After doing that i forgot to remove these lines.